packages/node-cli/src/templates/unittest/package.json.template

```diff
@@ -5,7 +5,8 @@
     "test": "jest"
   },
   "devDependencies": {
-    "jest": "^27.4.7",
+    "jest": "^29.4.3",
+    "@types/jest": "^29.4.0",
```

Comment:
@karl-anthony-ng according to npmjs this is the latest version they have
https://www.npmjs.com/package/@types/jest

Reply:
I think it is recommended but not mandatory.
See [this post in Stackoverflow](https://stackoverflow.com/questions/68003069/will-there-be-a-types-jest-version-27#:~:text=For%20%40types%2F*%20modules%20it's,%40types%2Fjest%20is%20ideal.)

However I would also note it would be important to not put the "^" as we prefer to have always this actual version of the libraries for future testing and finding the reason of issues.